### ISSUE TITLE
Add Snippet Trigger 🍺

### ISSUE THREAD

**Issue by deanishe** [jsumners/alfred-emoji#21]
A match made in heaven, tbh.

![snippet-trigger](https://user-images.githubusercontent.com/747913/34900717-9a19c1ce-f802-11e7-97d4-5c1733ef6768.gif)

Usually works best with an extra flag to set "paste" as default action.

**Comment by jsumners** [comment]
Are you submitting a PR for this?

**Comment by deanishe** [comment]
Nope. It's just a feature suggestions/request.

I just quickly added a Snippet Trigger to the my copy of the workflow via Alfred Preferences.

A proper implementation would require several new elements adding to default to the "paste" action. Given my preference for editing info.plist via Alfred, your dislike of that, and my lack of understanding of the way your build process works, I am not confident that I could submit a PR that would be acceptable.

**Comment by jsumners** [comment]
A thorough detail of what is necessary would be helpful. Or maybe a link to one that does the equivalent task.

**Comment by deanishe** [comment]
I've only added a Snippet Trigger to [one of my own workflows](https://github.com/deanishe/alfred-relative-dates) to date.

So, I think what you want is this:

![snippet-trigger](https://user-images.githubusercontent.com/747913/34901651-9304093e-f80d-11e7-838b-6487a137e1f6.png)

The user types the Snippet Trigger's snippet; Alfred runs the duplicate `emoji` Script Filter, and then pastes the selected emoji where the snippet was (the original Script Filter defaults to copy, not paste).

In the above-linked workflow, I used workflow variables to direct the execution flow (and avoid the Script Filter duplication), but that's not worth the effort here, imo, as you'd have to rewrite your Alfred feedback code to support passing [workflow variables](https://www.alfredforum.com/topic/9070-how-to-workflowenvironment-variables/).
